### PR TITLE
Skip osx fortran compiler version 4

### PR DESCRIPTION
**Pull request by ocefpaf**
~~@conda-forge/summa team, this should be removed once the code compiles with with `osx_fortran_compiler_version7`.~~

We actually need to skip 4 and not 7! So we are future proof here, if this works ;-p

### CODE REVIEW

**Comment by ocefpaf** [comment]
@isuruf I followed your suggestion from  https://github.com/conda-forge/staged-recipes/pull/5069#issuecomment-536027421, do you know what I did wrong here?

**Comment by isuruf** [comment]
Issue was that LDFLAGS were not set in gfortran_osx-64 package. Can you send a PR to https://github.com/conda-forge/gfortran_osx-64-feedstock/blob/master/recipe/activate-gfortran.sh#L87 copying lines from https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/master/recipe/activate-clang.sh#L89-L90 ?

**Comment by ocefpaf** [comment]
> Issue was that LDFLAGS were not set in gfortran_osx-64 package.

Good to know.

> Can you send a PR to [conda-forge/gfortran_osx-64-feedstock:recipe/activate-gfortran.sh@`master`#L87](https://github.com/conda-forge/gfortran_osx-64-feedstock/blob/master/recipe/activate-gfortran.sh#L87) [](/conda-forge/gfortran_osx-64-feedstock/blob/HEAD@{2019-10-01T16:22:32Z}/recipe/activate-gfortran.sh#L87) copying lines from [conda-forge/clang-compiler-activation-feedstock:recipe/activate-clang.sh@`master`#L89-L90](https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/master/recipe/activate-clang.sh#L89-L90) [](/conda-forge/clang-compiler-activation-feedstock/blob/HEAD@{2019-10-01T16:22:32Z}/recipe/activate-clang.sh#L89-L90) ?

On it!

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+fortran_compiler:              # [unix or win64]
+  - gfortran                   # [(linux64 or osx)]
+  - gfortran                   # [aarch64 or ppc64le or armv7l]
+fortran_compiler_version:      # [unix or win64]
+  - 7                          # [linux64 or osx or aarch64]
+  - 8                          # [ppc64le or armv7l]
```
Comment (isuruf) [inline recipe/conda_build_config.yaml:6]: You should remove this file after https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/286

Comment (CJ-Wright) [reply recipe/conda_build_config.yaml:6]: Sorry I'm confused, are we removing the `conda_build_config.yaml` file? Could I have more context?

Comment (isuruf) [reply recipe/conda_build_config.yaml:6]: We are not building with gfortran 4 after that PR is merged, so there's nothing to skip.

**Pull request merged**